### DISCUSSION TITLE
Hotend warmup during ABL

### DISCUSSION THREAD

**Discussion by JoeMaxey** [mriscoc/Ender3V2S1#86] [General]
How can I keep the hotend from warming up during the ABL process? I warm my bed up first, but not my hotend. I don't like that it warms up before starting the ABL and stays at printing temp. Would prefer to warm the hotend up after ABL completes.

**Comment by mriscoc** [comment]
Only the Manual mesh version warms up the hotend (and bed). BLTouch version only warms the bed and it is controllable under Advanced/Probe Settings.

**Reply by broderp1** [reply]
> Only the Manual mesh version warms up the hotend (and bed). BLTouch version only warms the bed and it is controllable under Advanced/Probe Settings.

How hard would it be to make it warm up the hotend as well? Seems like you would want everything at working temp to account for variances in manufacture and  wear.  If not why does the manual leveling have both heating up?

**Reply by fuzelet** [reply]
The manual version uses the hotend as the "probe" so it needs to be heated to make sure your manual measurements are accurate since the hotend is metal and expands with heat. Automatic versions use a probe, so whether the hotend is hot or cold has no effect on the probe.

**Reply by mriscoc** [reply]
In addition, the nozzle gets hot on the manual mesh version because it may have filament residues on the tip that interferes with using the nozzle as a probe.
